components/bms/bms_master/Core/Inc/selftest.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file selftest.h
+ * @brief Header file for BMS startup self tests
+ *
+ * @date 2020/10/03
+ * @author
+ */
+
+#ifndef INC_SELFTEST_H_
+#define INC_SELFTEST_H_
+
+#include "ltc6813_btm.h"
+
+/*============================================================================*/
+/* FUNCTION PROTOTYPES */
+BTM_Status_t ST_checkLTCtemp(void);
+BTM_Status_t ST_checkOpenWire(void);
+BTM_Status_t ST_checkOverlapVoltage(void);
+BTM_Status_t ST_checkVREF2(void);
+BTM_Status_t ST_verifyDischarge(BTM_PackData_t* pack);
+
+/*============================================================================*/
+/* CONFIGURABLE PARAMETERS */
+
+// ANDREW: To avoid overwriting constants, I'd suggest prefixing any constants
+// in the header file here with "ST_". Just don't forget to change them where
+// they're used if you do this!
```

Comment:
We should probably remove all of my comments with `// ANDREW:` from the code files to keep things clean; in the future I will make comments on Github like this one